python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -436,6 +436,11 @@ def run(
             ``outputCatalog`` : `pyarrow.Table`
                 Catalog with fit residuals of all sources used.
         """
+        if (len(inputVisitSummaries) == 1) and (self.config.deviceModel and self.config.exposureModel):
```

Comment:
You could skip the set of parenthesis towards the end of the line.
https://developer.lsst.io/python/style.html#the-minimum-number-of-parentheses-needed-for-correctness-and-readability-should-be-used

Reply:
True, I was thinking about `self.config.deviceModel` and `self.config.exposureModel` going together, but I realize it's logically the same without parentheses.